### DISCUSSION TITLE
Can each window host a different application in it?

### DISCUSSION THREAD

**Discussion by jfbaro** [nextapps-de/winbox#156] [Ideas]
Hello, the project looks GREAT! Congrats.

Can I consider each of those windows could hold a different application (different URL, maintained by a different team?)

Thanks

**Comment by jfbaro** [comment]
Hello there

We have conducted some tests, and it turns out that each window in WinBox.js behaves like an iframe with its own DOM, which is perfect for our use case. We're looking to provide a "Smart Desktop" to our employees (VPN), allowing them to access authorized applications according to their AD credentials.

To overcome the X-Frame limitation when embedding SaaS products into WinBox.js, we'll install this [Chrome extension](https://chrome.google.com/webstore/detail/ignore-x-frame-headers/gleekbfjekiniecknbkamfmkohkpodhe).

Just to clarify, this iframe solution would not work for public-facing websites, as we won't have control over the browser and its extensions that users might use to access our site. But for our use case, this is not a problem.

Thanks!

**Comment by ts-thomas** [comment]
Hello, thanks for the praise. It is a limitation by the browser, therefore it can't be solved by Winbox. It just can be solved by browser settings or extensions as you suggested (as far as I know). This only applies when using iFrame (extern hosted content on different origin). When you have an idea for a solution which could applied by Winbox please let me know.
